src/vmm/src/lib.rs

```diff
@@ -57,6 +57,12 @@ const ZEROPG_START: u64 = 0x7000;
 /// Address where the kernel command line is written.
 const CMDLINE_START: u64 = 0x0002_0000;
 
+/// Default high memroy start.
+pub const HIGH_RAM_START: u64 = 0x100000;
+
+/// Default kernel command line.
```

Comment:
nit: probably not in the scope of this PR but it would be nice to document a bit why we chose this default cmdline (or just open an issue for that).

Reply:
I am sort of starting to believe that the command line should not be exposed to the user in any way because it is very hard to configure it correctly. Instead of being able to configure the command line, we should allow users to configure options in a higher level language. For example, the typical usage of the command line is to disable the serial console device. Instead of allowing modification on the kernel command line, we can just add a flag parameter (`--serial`) and update the cmdline under the hood without direct user interaction. Does that make sense?

Reply:
Sounds nice.